packages/slate/test/models/node/get-closest/matching-current-using-path.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx h */
+
+import h from '../../../helpers/h'
+
+export const input = (
+  <value>
+    <document>
+      <quote key="a" {...{ className: 'slate-node' }}>
```

Comment:
Why write these like this with the object spreading? Instead of just writing `className=`? Even better might be to use `thing="value"` or something more abstract just to avoid confusion with DOM stuff.